tests/x11/tracker/tracker_by_command.pm

```diff
@@ -16,12 +16,11 @@ use base "x11test";
 use strict;
 use testapi;
 use utils;
-use version_utils 'is_sle';
-
+use version_utils qw(is_sle is_opensuse);
 
 sub run {
     x11_start_program('xterm');
-    if (is_sle('12-SP2+')) {
+    if (is_opensuse || is_sle('12-SP2+')) {
```

Comment:
looks ok but not entirely correct as "opensuse" does not make the difference but package versions which we try to model within product versions as you know. So how about `if (!is_sle('<12-SP2'))` ?